laboratory-service/src/main/java/com/raf/si/laboratoryservice/dto/request/CreateReferralRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.raf.si.laboratoryservice.dto.request;
+
+import com.raf.si.laboratoryservice.model.enums.referral.ReferralType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CreateReferralRequest {
+    @NotNull(message = "Polje uput ne sme biti prazno")
```

Comment:
Nemoj u requestovima korisitit enume, jer se ne prikazuju tako na frontu. Uzmi ih kao string, pa proveri da li enum za tu notaciju postoji na beku. Mozes da pogledas patientMapper da vidis kako smo to radili